client/src/layers/EditAnnotationLayer.ts

```diff
@@ -45,6 +45,10 @@ export default class EditAnnotationLayer extends BaseLayer<GeoJSON.Feature> {
 
   hoverHandleIndex: number;
 
+  disableModeSync: boolean;
+
+  leftButtonCheckTimeout: number;
```

Comment:
Please add inline explanations of what these are for.  You explained it a bit in the github issue, but that information would be more helpful here.

Also, if you think this a hack and want to fix it, perhaps a `// TODO reevaluate this solution` or something would be appropriate.  Don't want to lose track of bad code if you think this isn't a good permanent fix.